doc/version_history.rst

```diff
@@ -6,6 +6,13 @@
 Version History
 ###############
 
+v1.1.0
+------
+
+* Use ts_pre_commit_conf.
+* Fix an incorrect argument name in a method doc string.
+* Add an abstract subclass of OneClientServer that includes a generic read loop.
```

Comment:
Please also mention the new methods on BaseClientOrServer (and thus on Client and OneClientServer)